### PR TITLE
holiday self-approval check list

### PR DESCRIPTION
**Pull request by peterlangmar**
There were holidays, which were not comforting the work of the other partners. The reason was the lack of detailed check-list

### CODE REVIEW

**Comment by EggDice** [comment]
I would add, that the velocity loss is negotiated with the client.

**Comment by EggDice** [comment]
Also replacement resources are on-boarded to the project.

File: README.md [hunk 1]
```diff
@@ -559,13 +559,20 @@ Policy proposals longer than 10 sentences must be shared at least 48h before the
 ##### Policy: Asana for top level boards
 Use [Asana](https://app.asana.com/0/31935013151061) for top level circle boards
 
-##### Policy: Holidays / Home office / Sick leave notification and approval
-You have to clearly distinguish between holidays, home office and sick leave. Mark it in your calendar as soon as you're aware of it.
-- Holidays: Let your circles (on Slack) and your clients know your planned holiday not later then 2x workdsays of its length, but minimum 5 workdays.
-- Home office: Let your circles on Slack know on the previous day if you would like to work from home all day long.
-In case of Holidays and Home office, letting them know within this period doesn't mean that it's not allowed, but  can be declined, if your plan is not safe to fail.
+##### Policy: Holidays / Learning Time / Home office / Sick leave notification and approval
+- Clearly distinguish between Holidays / Learning Time / Home office / Sick leave
```
Comment (cassus) [inline README.md]: Clearly distinguish in my calendar? Or when I talk about it?

File: README.md [hunk 2]
```diff
@@ -559,13 +559,20 @@ Policy proposals longer than 10 sentences must be shared at least 48h before the
...
+- Mark everything in your calendar as soon as you're aware of it.
+- Holidays / Learning Time self-approval check-list to make your absence safe
```
Comment (cassus) [inline README.md]: What counts as learning time? Do I have to ask permission from Resource Clarity if I go to a meetup?

File: README.md (hunk 1 above)
Comment (peterlangmar) [inline README.md]: both

File: README.md (hunk 2 above)
Comment (peterlangmar) [inline README.md]: self-approval list, so this is not specified
if you have a tension, you can add further specifications to this policy later on

**Comment by peterlangmar** [comment]
@EggDice "Also replacement resources are on-boarded to the project." what does this mean?

**Comment by peterlangmar** [comment]
@EggDice how would you "Also replacement resources are on-boarded to the project."?

**Comment by EggDice** [comment]
I meant if you find somebody to do your job till you are on holiday, you should show the exact processes, and tell exactly what to do and what did you agree with the client before you go.

**Comment by peterlangmar** [comment]
@EggDice incorporated

**Comment by EggDice** [comment]
:+1:

File: README.md (hunk 2 above)
Comment (cassus) [inline README.md]: :+1:

File: README.md (hunk 1 above)
Comment (cassus) [inline README.md]: Having this specificity in the text would help me align with it

**Comment by cassus** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (peterlangmar) [inline README.md]: @cassus amended

**Pull request merged**